src/main/sensors/gyro.c

```diff
@@ -549,6 +551,9 @@ bool gyroInit(void)
 
     bool ret = false;
     gyroToUse = gyroConfig()->gyro_to_use;
+    
+    gyroForAxis[3] = gyroConfig()->gyro_for_axis[3];
```

Comment:
`for` loop must be used for copy

Reply:
Do you mean create 3 variables and copy them into gyroForAxis[3]?

Reply:
```
for (int i =0; i < XYZ_AXIS_COUNT; i++)
   gyroForAxis[i] = gyroConfig()->gyro_for_axis[i];